src/lib/pickles/wrap_verifier.ml

```diff
@@ -473,14 +497,16 @@ struct
             | Opt.Nothing ->
                 None
             | Opt.Maybe (keep, p) ->
+                Array.iter ~f:Point.assert_finite p ;
                 Some
-                  { non_zero = Boolean.(keep &&& Point.finite p)
-                  ; point = Point.underlying p
+                  { non_zero = Boolean.(keep &&& true_)
```

Comment:
Same than above. Why?

Reply:
`Point.finite p = false_` is unreachable, so we hard-code `true_`